src/meta/meta_bulk_load_service.cpp

```diff
@@ -1120,6 +1143,41 @@ void bulk_load_service::update_app_status_on_remote_storage_reply(const app_bulk
     }
 }
 
+// ThreadPool: THREAD_POOL_META_STATE
+bool bulk_load_service::check_ever_ingestion_succeed(const partition_configuration &config,
+                                                     const std::string &app_name,
+                                                     const gpid &pid)
+{
+    partition_bulk_load_info pinfo;
```

Comment:
partition_bulk_load_info *pinfo;
    {
        zauto_read_lock l(_lock);
        pinfo = &_partition_bulk_load_info[pid];
    }

Reply:
Why use * ？

Reply:
In this function, `_partition_bulk_load_info[pid]` won't be updated. I think it's okay to use the value copy here.